File: core/src/main/java/com/function/epub/model/BookInfoData.java
package com.function.epub.model;

import com.core.base.BaseModel;

public class BookInfoData extends BaseModel {
    private static final BookInfoData ourInstance = new BookInfoData();

    public static BookInfoData getInstance() {
        return ourInstance;
    }

    private BookInfoData() {
    }

    private BookInfo bookInfo;

    public BookInfo getBookInfo() {
        return bookInfo;
    }

    public void setBookInfo(BookInfo bookInfo) {
        this.bookInfo = bookInfo;
    }
}

File: core/src/main/java/com/loitp/func/epub/model/BookInfoData.kt
package com.loitp.func.epub.model

import androidx.annotation.Keep
import com.loitp.core.base.BaseModel

/**
 * Created by Loitp on 04,August,2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
@Keep
class BookInfoData private constructor() : BaseModel() {

    companion object {
        val instance = BookInfoData()
    }

    var bookInfo: BookInfo? = null
}
